rector-ci.php

```diff
@@ -76,6 +78,7 @@
         SplitStringClassConstantToClassConstFetchRector::class,
         // false positives on constants used in rector-ci.php
         RemoveUnusedClassConstantRector::class,
+        MoveOutMethodCallInsideIfConditionRector::class,
```

Comment:
This rule should be enabled. If there are some bugs, it should be fixed.

I'll merge this as it is, so it can be handled in standalone PR.